tests/test_suggestions.py

```diff
@@ -0,0 +1,16 @@
+from pytube import Search
+#might have to handle new objects that might be created by YouTube in the future
+def test_completion_suggestions():
+    temp = "#crapsnation" #handled missing hashtagTileRenderer object
+    s = Search(temp)
+    assert s.completion_suggestions
+
+    temp_1 = "No Time To Die Billie Eilish" #handled missing movieRenderer object
+    s = Search(temp_1)
+    assert s.completion_suggestions
+
+    temp_2 = "rangmnam style" #handled missing showingResultsForRenderer object
+    s = Search(temp_2)
+    assert s.completion_suggestions
+
+test_completion_suggestions()
```

Comment:
Do not call the function yourself. Read the basics of [pytest documentation](https://docs.pytest.org/en/latest/getting-started.html) for more information.

Reply:
+1

also assert "s.completion_suggestions" is asserting the boolean quantity of the object which is a bit difficult to parse at first read -- generally easy to assert notEmpty or similar

Reply:
I'm currently working on changing this, thank you for your feedback @RONNCC I will get back to you asap.

Reply:
@glubsy I'm trying to implement the code using mocks as you suggested, I've stored 3 different .html files (for the cases I've implemented the fix) in the mock folder. I'm trying to directly access these .html files using my test code for simulating the requests. Could you please provide your advice on this and confirm if I'm heading in the right direction? After completing my testing on the mock folder, do I zip it? Thanks in advance!

Reply:
Let's keep this discussion in the main thread instead.